py/src/braintrust/framework.py

```diff
@@ -733,7 +749,9 @@ async def run_evaluator_task(datum):
 
                 # Check if the task takes a hooks argument
                 task_args = [datum.input]
-                if len(inspect.signature(evaluator.task).parameters) == 2:
+                if not is_inspectable(evaluator.task):
+                    task_args.append(hooks)
+                elif len(inspect.signature(evaluator.task).parameters) == 2:
```

Comment:
maybe `or` these together since they do the same thing?

Reply:
Sure.